src/Repository/LineItemRepository.php

```diff
@@ -48,4 +60,16 @@ public function findAllAsCollection(): LineItemCollection
 
         return $collection;
     }
+
+    public function findById(int $id): ?LineItem
```

Comment:
I'm consistent. IMO find methods should NOT return this exception unless you specify it on the method name. This shouldn't be an exception because it's expected to do not have such information on the database. If you take a look at the `find` or `findOneBy` Methods, they do not return exceptions.

Reply:
Let's fix it based on our discussion yesterday